src/components/top-bar/styles.ts

```diff
@@ -0,0 +1,69 @@
+import { styled } from '@linaria/react'
+import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../styles/media'
+import { ElButtonGroup } from '../button-group'
+import { css } from '@linaria/core'
+
+export const ElTopBarProfile = styled.div`
+  padding: var(--spacing-1, 4px) var(--spacing-none, 0px);
+`
+
+export const ElTopBarMainNav = styled(ElButtonGroup)`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding-left: var(--spacing-6, 24px);
+  flex-grow: 1;
+`
+
+export const ElTopBarSecondaryNav = styled(ElButtonGroup)`
+  flex-wrap: nowrap;
+  padding-right: var(--spacing-spacing-2, 8px);
```

Comment:
@nurm717123 
**issue:** Same issue with the naming convention for the variable.

Reply:
Ahh, good catch @ksolanki7 🙌